packages/react-native-reanimated/apple/reanimated/apple/REAModule.mm

```diff
@@ -327,6 +304,7 @@ - (void)commonInit:(std::shared_ptr<ReanimatedModuleProxy>)reanimatedModuleProxy
   WorkletRuntimeCollector::install(rnRuntime);
   RNRuntimeDecorator::decorate(rnRuntime, reanimatedModuleProxy);
 #ifdef RCT_NEW_ARCH_ENABLED
+  [self attachReactEventListener];
```

Comment:
Why do we need this change here?

Reply:
We'd call it on Fabirc before `commonInit` here https://github.com/software-mansion/react-native-reanimated/pull/6783/files#diff-06f7028a5f8a14ac525b70098bbf48937abe91707aea813bc4528ec9c166f5bcL296 so I moved it into `commonInit` fabric section.